include/wx/gtk/glcanvas.h

```diff
@@ -111,6 +111,8 @@ class WXDLLIMPEXP_GL wxGLCanvas : public wxGLCanvasX11
     // implementation from now on
     void OnInternalIdle() wxOVERRIDE;
 
+    void EnsureAvailable();
```

Comment:
Not sure if we want to expose this hack in the header, this could be a private function in `src/gtk/glcanvas.cpp` just as well AFAICS.